src/screencapturekit.cs

```diff
@@ -335,4 +453,105 @@ interface SCStreamOutput {
 		[Export ("stream:didOutputSampleBuffer:ofType:")]
 		void DidOutputSampleBuffer (SCStream stream, CMSampleBuffer sampleBuffer, SCStreamOutputType type);
 	}
+
+	[NoiOS, NoTV, NoWatch, Mac (14, 0), NoMacCatalyst]
+	[BaseType (typeof (NSObject))]
+	interface SCContentSharingPickerConfiguration {
+		[Export ("allowedPickerModes", ArgumentSemantic.Assign)]
+		SCContentSharingPickerMode AllowedPickerModes { get; set; }
+
+		[Export ("excludedWindowIDs", ArgumentSemantic.Strong)]
+		NSNumber [] ExcludedWindowIDs { get; set; }
+
+		[Export ("excludedBundleIDs", ArgumentSemantic.Strong)]
+		string [] ExcludedBundleIDs { get; set; }
+
+		[Export ("allowsChangingSelectedContent")]
+		bool AllowsChangingSelectedContent { get; set; }
+	}
+
+	[NoiOS, NoTV, NoWatch, Mac (14, 0), NoMacCatalyst]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface SCContentSharingPicker {
+		[Static]
+		[Export ("sharedPicker")]
+		SCContentSharingPicker SharedPicker { get; }
+
+		[Export ("defaultConfiguration", ArgumentSemantic.Copy)]
+		SCContentSharingPickerConfiguration DefaultConfiguration { get; set; }
+
+		[NullAllowed]
+		[Export ("maximumStreamCount", ArgumentSemantic.Strong)]
+		NSNumber MaximumStreamCount { get; set; }
+
+		[Export ("active")]
+		bool Active { [Bind ("isActive")] get; set; }
+
+		[Export ("addObserver:")]
+		void AddObserver (SCContentSharingPickerObserver observer);
+
+		[Export ("removeObserver:")]
+		void RemoveObserver (SCContentSharingPickerObserver observer);
+
+		[Export ("setConfiguration:forStream:")]
+		void SetConfiguration ([NullAllowed] SCContentSharingPickerConfiguration pickerConfig, SCStream stream);
+
+		[Export ("present")]
+		void Present ();
+
+		[Export ("presentPickerUsingContentStyle:")]
+		void PresentPickerUsingContentStyle (SCShareableContentStyle contentStyle);
+
+		[Export ("presentPickerForStream:")]
+		void PresentPickerForStream (SCStream stream);
+
+		[Export ("presentPickerForStream:usingContentStyle:")]
+		void PresentPickerForStream (SCStream stream, SCShareableContentStyle contentStyle);
+	}
+
+	[NoiOS, NoTV, NoWatch, Mac (14, 0), NoMacCatalyst]
+	[Protocol]
+	[BaseType (typeof (NSObject))]
+	interface SCContentSharingPickerObserver {
```

Comment:
You are missing the `[Model]` attribute here the rule is

Is a `[Protocol]` ? You need both `[Model]` and `[BaseType]`

```suggestion
	[NoiOS, NoTV, NoWatch, Mac (14, 0), NoMacCatalyst]
	[Protocol][Model]
	[BaseType (typeof (NSObject))]
	interface SCContentSharingPickerObserver {
```